templates/caching-service.json

```diff
@@ -314,32 +363,62 @@
       {
          "description": "Namespace for the application. Note: The namespace is required for creating proper RoleBindings. Specifying wrong namespace will prevent cluster from forming.",
          "name": "NAMESPACE",
+         "displayName": "Namespace",
          "required": true,
          "value": "myproject"
       },
       {
          "description": "The name for the application.",
          "name": "APPLICATION_NAME",
+         "displayName": "Application Name",
          "required": true,
          "value": "caching-service-app"
       },
       {
          "description": "Infinispan image.",
          "name": "IMAGE",
          "required": true,
-         "value": "docker-registry.engineering.redhat.com/jboss-dataservices/datagrid-online-services"
+         "value": "172.30.1.1:5000/myproject/datagrid-online-services-dev"
       },
       {
          "description": "Number of instances in the cluster.",
          "name": "NUMBER_OF_INSTANCES",
+         "displayName": "Number of Instances",
          "required": true,
          "value": "1"
       },
       {
          "description": "Total container memory in MiB.",
+         "displayName": "Total Memory",
          "name": "TOTAL_CONTAINER_MEM",
          "required": false,
          "value": "512"
+      },
+      {
+         "name": "APPLICATION_USER",
+         "displayName": "Client User",
+         "description": "Username for client applications",
+         "required": true
+      },
+      {
+         "name": "APPLICATION_USER_PASSWORD",
+         "displayName": "Client Password",
+         "description": "Password for client applications",
+         "generate": "expression",
+         "from": "[a-zA-Z0-9]{16}"
+      },
+      {
+         "name": "MANAGEMENT_USER",
+         "displayName": "Management User",
+         "description": "Username for performing management operations",
+         "required": false
+      },
+      {
+         "name": "MANAGEMENT_USER_PASSWORD",
```

Comment:
If you agree with me that management realm is not needed - we should remove this.